api/core/v1alpha1/prefix_types.go

```diff
@@ -77,3 +77,50 @@ func (in *IPPrefix) DeepCopy() *IPPrefix {
 	in.DeepCopyInto(out)
 	return out
 }
+
+func (in *IPPrefix) First() netip.Addr {
+	if !in.IsValid() {
+		return netip.Addr{}
+	}
```

Comment:
```suggestion
```
Based on the documentation for `Masked` this is not required.

https://pkg.go.dev/net/netip@go1.24.3#Prefix.Masked
> If p is zero or otherwise invalid, Masked returns the zero [Prefix](https://pkg.go.dev/net/netip@go1.24.3#Prefix).